src/generated/xinput.rs

```diff
@@ -733,45 +730,38 @@ impl InputInfoInfo {
             Some(result) => Ok((result, outer_remaining)),
         }
     }
-    pub fn as_key(&self) -> Option<&InputInfoInfoKey> {
-        match self {
-            InputInfoInfo::Key(value) => Some(value),
-            _ => None,
-        }
-    }
-    pub fn as_num_buttons(&self) -> Option<&u16> {
-        match self {
-            InputInfoInfo::NumButtons(value) => Some(value),
-            _ => None,
-        }
-    }
-    pub fn as_valuator(&self) -> Option<&InputInfoInfoValuator> {
-        match self {
-            InputInfoInfo::Valuator(value) => Some(value),
-            _ => None,
-        }
-    }
```

Comment:
I didn't even know these functions existed, heh. :-)
Can you reintroduce these? Or do you think they are useless and the caller should just `match` on its own.

Reply:
I think I introduced them in a PR. I will add them again.